www/js/goals/goals-login.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+angular.module('emission.main.goalslogin',['emission.services', 'ngSanitize', 'ngAnimate', 'angularLocalStorage'])
+
+/*.run(function(CommHelper, $state, ionicPlatform, $rootScope, storage) {
+	$ionicPlatform.ready(function() {
+	    var getUserInfo = function(){
+	        var callOpts = {'method': 'GET', 'method_url': "/api/v3/user",
+	                        'method_args': null};
+
+	        CommHelper.habiticaProxy(callOpts).then(function(response){
+	            console.log("Success!");
+	            console.log(response);
+	            $state.go('root.main.goals.goalhome');
+	        }, function(error) {
+	            console.log("Not signed up");
+	        });
+		};
+		getUserInfo();	
+    });
+})*/
+
+.controller("GoalsLoginCtrl", function(CommHelper, $state, $rootScope, storage, $ionicLoading, $ionicPopup) {
```

Comment:
This looks like it is a change for a new login screen, not to hide test users. Am I missing something here?